FileIO/GMSInterface.cpp

```diff
@@ -314,10 +319,23 @@ MeshLib::Mesh* GMSInterface::readGMS3DMMesh(const std::string &filename)
 	}
 
 	in.close();
-	INFO("GMSInterface::readGMS3DMMesh(): \tfinished.");
+	INFO("GMSInterface::readGMS3DMMesh(): finished.");
 
 	const std::string mesh_name = BaseLib::extractBaseNameWithoutExtension(filename);
-	return new MeshLib::Mesh(mesh_name, nodes, elements);
+	MeshLib::Properties properties;
+	if (mat_ids.size() == elements.size())
+	{
+		boost::optional<MeshLib::PropertyVector<int> &> opt_pv
+			= properties.createNewPropertyVector<int>(
+			"MaterialIDs", MeshLib::MeshItemType::Cell);
+		assert(opt_pv != boost::none);
```

Comment:
I don't think using assert is a good idea. If the programme is closed when I am testing complex workflow this is going to be really annoying. Even if it's only in Debug-mode.